tao/setup.py

```diff
@@ -33,12 +34,11 @@
 
 
 def detect_host_tf_version():
-    version = tf.__version__
```

Comment:
this never works well?

Reply:
the name of tensorflow wheel package changes on different platform (e.g. `tensorflow-aarch64`, `tensorflow-gpu`). We use a more formal way to query the wheel package name instead of explicitly enumerating the name.